docker/springboot-realm.json

```diff
@@ -381,7 +381,7 @@
     "enabled" : true,
     "clientAuthenticatorType" : "client-secret",
     "secret" : "f80d93b4-9039-4012-99c8-a45663485862",
-    "redirectUris" : [ "http://localhost:8080/*" ],
+    "redirectUris" : [ "http://localhost:8080/*", "http://localhost:8082/*"],
```

Comment:
This is used when you access a rest end point using the web browser and you want to be redirected to keycloak login page, then back to you application.